oap-server/server-starter/src/main/resources/oal/core.oal

```diff
@@ -25,8 +25,6 @@ service_sla = from(Service.*).percent(status == true);
 service_cpm = from(Service.*).cpm();
 service_percentile = from(Service.latency).percentile(10); // Multiple values including p50, p75, p90, p95, p99
 service_apdex = from(Service.latency).apdex(name, status);
-service_mq_consume_count = from(Service.*).filter(type == RequestType.MQ).count();
```

Comment:
I think these should be kept. As this is still meaningful from a service perspective.

Reply:
I  have restored it
![image](https://user-images.githubusercontent.com/3917424/198181357-89660653-d1c0-4b61-abad-8d1adb64feb4.png)

Reply:
I am afraid you didn't revert `general-endpoint` dashboard?

Reply:
endpoint_mq_consume_latency , endpoint_mq_consume_count should be demonstrated in  virtual mq  and  endpoint dashboard  at the same time ？

Reply:
The endpoint of the service is not the endpoint of the virtual queue service. This is not showing one thing in two places. There are two dimensions statistics, right?

Reply:
We provide virtual service to demonstrate special statistics , and should  keep `General-Endpoint` common (`General-Service` also is )

The following screenshots from my local env 

![image](https://user-images.githubusercontent.com/3917424/198235427-e0d0e4fb-55be-4ac1-b08c-1ba074046fae.png)

int  another aspect  , analyzing `endpoint of the service `  and  `endpoint of the virtual queue service` would produce repeat logic/calculation

Reply:
> int another aspect , analyzing endpoint of the service and endpoint of the virtual queue service would produce repeat logic/calculation

What do you mean by repeat?
Let's put it this way. Queue-A is consumed by serv1 and serv2 at the same time.
1. From the service perspective, serv1./queue-a is the entity of the metrics.
2. From the virtual MQ perspective, queue-a is an endpoint of the MQ service(IP based).

Those are different. I don't think this is a repeated calculation. The case it is duplicated only when a MQ only have one consumer service.

Reply:
Sorry,  It's two dimensional .
And  I  have another  question .

![image](https://user-images.githubusercontent.com/3917424/198246447-a7d961fa-009e-411a-83f3-2951ef46f2f7.png)

As  above screenshot , there is an endpoint named `Kafka/topic/Consumer/a` , in this scenario , `Endpoint Load (calls / min)
` is  similar with `Message Queue Consuming Count ` except  one is `CPM` , another is `Count ` .   Right ?

Reply:
I think those are really some kinds of duplicates. I didn't notice this.

It should be fine to remove MQ's consuming count, meanwhile, we should update the title and tips for Endpoint CPM metric to mention it has various meanings for different cases.

Reply:
And do we have another kind of `transmission.latency` ? 
I  think we should change
 `endpoint_mq_consume_latency = from((str->long)Endpoint.tag["transmission.latency"]).filter(type == RequestType.MQ).filter(tag["transmission.latency"] != null).longAvg();`    

to  

`endpoint_transmission_latency = from((str->long)Endpoint.tag["transmission.latency"]).filter(tag["transmission.latency"] != null).longAvg();`

Reply:
I think both are fine. The only difference is `.filter(type == RequestType.MQ)`, which is faster than not null. As in OAL compiling, `.filter(tag["transmission.latency"] != null)` means to read a value from a map, but the `type` is a simple `getFromField`.

Reply:
The  main purpose is to make the metrics as  `transmission.latency metrics ` rather than `mq  transmission.latency metrics ` , like above example  the endpoint name should distinguish whether it's  related with MQ

Reply:
Then I think endpoint_mq_consume_latency is better. Because it indicates MQ endpoint only.

Reply:
Yes , It's better if we only have MQ transmission latency , if we would provide other kind of `transmission latency` such as  `HTTP transmission latency`  in  future , `endpoint_transmission_latency ` should be  better . in order to make `transmission latency` more  flexible , I  remove `.filter(type == RequestType.MQ)`

Reply:
HTTP transmission latency means the client side has to inject the timestamp of the client side and calculate the duration on the server side. But different from MQ, HTTP latency is usually less than 5ms, then time sync between client and server matters a lot, in the real prod env, I don't believe any infra team could set this up.

So, this is why the `transmission latency` is specifically for MQ only.